### PR TITLE
Make links on our red background accessible

### PR DESCRIPTION
**Pull request by sjparkinson**
**Before**

image

**After**

image

### CODE REVIEW

File: src/scss/themes/_live.scss [hunk 1]
```diff
@@ -21,9 +21,17 @@
...
-        .o-teaser__heading a:visited {
-            color: oColorsGetPaletteColor('white-80');
+        .o-teaser__heading a {
+            &:focus,
+            &:hover {
+                color: oColorsMix(white, crimson, 90);
+            }
+
+            &:visited {
+                color: oColorsGetPaletteColor('white-80');
```
Comment (rowanmanning) [inline src/scss/themes/_live.scss]: Is "visited" still supposed to be `white-80`? Because `white-80` still has some FT pink in it

**Comment by sjparkinson** [comment]
screen recording 2018-11-15 at 14 57 44 1

The worst GIF I have ever made.

File: src/scss/themes/_live.scss (hunk 1 above)
Comment (sjparkinson) [reply src/scss/themes/_live.scss]: I don't know. I'm aware that it was a fix that Arjun added in https://github.com/Financial-Times/o-teaser/pull/98.

Comment (sjparkinson) [reply src/scss/themes/_live.scss]: "I probably completely made that up." – Arjun, November 2018

Comment (sjparkinson) [reply src/scss/themes/_live.scss]: We've updated visited to match hover and focus.

Comment (sjparkinson) [reply src/scss/themes/_live.scss]: We also noticed the imperceptible difference for the default style while looking into this.

https://github.com/Financial-Times/o-teaser/blob/5d2b6fa253612b94de53ce3f1780395b647182b3/src/scss/elements/_default.scss#L46-L53

**Review by rowanmanning — APPROVED** [review]
Gonna approve in Gabi and Lee's absence, but not super confident with o-teaser. Are you happy to merge and release, Sam?

**Comment by sjparkinson** [comment]
Yep 👌

**Pull request merged**